src/udev/udev-builtin-path_id.c

```diff
@@ -663,12 +663,6 @@ static int builtin_path_id(struct udev_device *dev, int argc, char *argv[], bool
                         path_prepend(&path, "xen-%s", udev_device_get_sysname(parent));
                         parent = skip_subsystem(parent, "xen");
                         supported_parent = true;
-                } else if (streq(subsys, "virtio")) {
-                        while (parent && streq_ptr("virtio", udev_device_get_subsystem(parent)))
-                                parent = udev_device_get_parent(parent);
-                        path_prepend(&path, "virtio-pci-%s", udev_device_get_sysname(parent));
-                        supported_transport = true;
```

Comment:
I just realized that revert alone won't do, because we must have supported_transport set to true. I'll respin the branch.